qutebrowser/config/configdata.yml

```diff
@@ -632,22 +641,50 @@ content.host_blocking.lists:
 
     The file `~/.config/qutebrowser/blocked-hosts` is always read if it exists.
 
-content.host_blocking.whitelist:
+content.blocking.method:
+  default: auto
+  type:
+    name: String
+    valid_values:
+      - auto: "Use Brave's ABP-style adblocker if available, host blocking
+        otherwise"
+      - adblock: Use Brave's ABP-style adblocker
+      - hosts: Use hosts blocking
+      - both: Use both hosts blocking and Brave's ABP-style adblocker
+  desc: |
+    Which method of blocking ads should be used.
+
+    Support for Adblock Plus syntax blocklists requires the `adblock` Python
+    package to be installed, which is an optional dependency of qutebrowser.
+    It can be installed using the command `pip install --user adblock`. This
```

Comment:
Yeah, I was hoping someone would make an AUR submission. I'm not entirely sure if I'll be able to—is it enough to install `python-adblock`, or does one need to take additional steps to install (some) rust depencies as well?

Reply:
I'd hope someone would package it for `[community]` - I've pinged @Foxboron (who packages qutebrowser) and he pinged some Rust packaging people :slightly_smiling_face: 

> I think I've answered my own question. The pip package comes with an .so file.

Yes, though a proper package (AUR or not) would build `python-adblock` from source rather than using the binary wheel directly.

Reply:
tl;dr: `python-adblock` is entering `[community]` when I allocate some time for it. Probably the weekend?

Reply:
Take your time - it'll probably still take a bit until this is merged and a new qutebrowser release with it is out.